### PR TITLE
Change table for podcasts

### PR DESCRIPTION
**Pull request by EugeniusGR**
image

### CODE REVIEW

**Comment by EugeniusGR** [comment]
mb increase margin-top?

File: frontend/src/components/podcast/components/episode-table/helpers/get-columns.helper.ts [hunk 1]
```diff
@@ -4,15 +4,15 @@ const getColumns = (): Column[] => {
   const columns: Column[] = [
     {
       Header: '#',
-      accessor: (_originalRow, rowIndex): string => String(rowIndex),
+      accessor: (_originalRow, rowIndex): string => String(rowIndex + 1),
```
Comment (ghost) [inline frontend/src/components/podcast/components/episode-table/helpers/get-columns.helper.ts]:
```suggestion
      accessor: (_originalRow, rowIndex): number => rowIndex + INCREASE_CONT_FOR_IDX,
```

File: frontend/src/hooks/hooks.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ export { useSelector, useDispatch } from 'react-redux';
 export { useForm } from 'react-hook-form';
 export { useHistory, useLocation, useParams } from 'react-router-dom';
 export { useTable } from 'react-table';
+export { format } from 'date-fns';
```
Comment (ghost) [inline frontend/src/hooks/hooks.ts]:
```suggestion
```

File: frontend/src/components/podcast/components/episode-table/helpers/get-rows.helper.ts [hunk 3]
```diff
@@ -1,9 +1,11 @@
 import { Episode } from 'common/types/types';
+import { format } from 'hooks/hooks';
```
Comment (ghost) [inline frontend/src/components/podcast/components/episode-table/helpers/get-rows.helper.ts]:
```suggestion
```

File: frontend/src/components/podcast/components/episode-table/helpers/get-rows.helper.ts [hunk 4]
```diff
@@ -1,9 +1,11 @@
...
 
 const getRows = (episodes: Episode[]): Record<string, string>[] => {
   const rows = episodes.map((episode) => ({
     name: episode.name,
     description: episode.description,
+    createdAt: format(new Date(episode.createdAt), 'dd/MM/yyyy'),
```
Comment (ghost) [inline frontend/src/components/podcast/components/episode-table/helpers/get-rows.helper.ts]: use helper for it

File: frontend/src/hooks/hooks.ts (hunk 2 above)
Comment (ghost) [reply frontend/src/hooks/hooks.ts]: we don't have this package on frontend

**Review by ghost — CHANGES_REQUESTED** [review]

File: frontend/src/hooks/hooks.ts (hunk 2 above)
Comment (EugeniusGR) [reply frontend/src/hooks/hooks.ts]: should I install it?

File: frontend/src/components/podcast/components/episode-table/helpers/get-columns.helper.ts [hunk 5]
```diff
@@ -1,18 +1,20 @@
 import { Column } from 'react-table';
+import { INCREASE_CONT_FOR_IDX } from 'jabber-shared/common/constants/constants';
```
Comment (ghost) [inline frontend/src/components/podcast/components/episode-table/helpers/get-columns.helper.ts]: we can't import from the shared directly

File: frontend/src/components/podcast/components/episode-table/helpers/get-columns.helper.ts [hunk 6]
```diff
@@ -1,18 +1,20 @@
...
 
 const getColumns = (): Column[] => {
   const columns: Column[] = [
     {
       Header: '#',
-      accessor: (_originalRow, rowIndex): string => String(rowIndex),
+      accessor: (_originalRow, rowIndex): number =>
+        rowIndex + INCREASE_CONT_FOR_IDX,
```
Comment (ghost) [inline frontend/src/components/podcast/components/episode-table/helpers/get-columns.helper.ts]:
```suggestion
      accessor: (_originalRow, rowIndex): number => {
        return rowIndex + INCREASE_CONT_FOR_IDX
      }
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**